run_autoformat.py

```diff
@@ -43,6 +43,7 @@ def _run_task_for_extension(extension, task):
         ('.scala', 'format-scala'),
         ('.py', 'format-python'),
         ('.json', 'format-json'),
+        ('.csv', 'check-reference_data'),
```

Comment:
Does this mean that files get copied across as part of the linting phase? It makes sense to check them, but I'm not sure about copying them...

Reply:
The current script automatically copies them across, but I think we’re all agreed that we don’t actually want that.